fact_extractor/plugins/unpacking/draytek_vigor_167/test/test_draytek_vigor_167.py

```diff
@@ -12,17 +12,18 @@ def test_unpacker_selection_generic(self):
         self.check_unpacker_selection('firmware/draytek-vigor-167', 'Draytek Vigor 167')
 
     def test_extraction(self):
```

Comment:
There is a little bit too much going on in this test case.

Can we please split it into two tests that are called something around the lines of `*_valid_image` and  `*_invalid_image`?